platform/security/pdp/security-pdp-authzrealm/pom.xml

```diff
@@ -83,6 +83,16 @@
             <groupId>ddf.platform.util</groupId>
             <artifactId>platform-util</artifactId>
         </dependency>
+        <dependency>
+            <groupId>oro</groupId>
+            <artifactId>oro</artifactId>
+            <version>2.0.8</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.servicemix.bundles</groupId>
+            <artifactId>org.apache.servicemix.bundles.commons-validator</artifactId>
```

Comment:
Shouldn't have a dependency on the servicemix bundle. This should be a dependency on the commons-validator jar instead.